docs/book/src/plugins/multi-module-v1-alpha.md

```diff
@@ -0,0 +1,77 @@
+# Multi Module Plugin (multi-module/v1-alpha)
+
+The multi-module plugin allows users to modify the existing scaffolding setup of kubebuilder to enable [Monorepo][monorepo] support. It abstracts the complexities to achieve this goal by easing the most difficult part of getting into Monorepos, namely the setup and the scaffolding required.

```

Comment:
Would make sense if we call the plugin such as monorepo instead of multi-module?

Reply:
Since the go documentation uses the terminology `multi-module` I think this is more fitting: https://go.dev/doc/tutorial/workspaces I can adjust the readme on this occasion